Packs/CSCDomainManager/Integrations/CSCDomainManager/CSCDomainManager.py

```diff
@@ -0,0 +1,553 @@
+
+    hr_output = get_domains_availability_check_hr_fields(available_domains_results)
+
+    results = CommandResults(
+        readable_output=tableToMarkdown('Domains Availability', hr_output, headers=HR_HEADERS_FOR_AVAILABILITY),
+        outputs_prefix='CSCDomainManager.Domain.Availability',
+        outputs=available_domains_results
+    )
+    return results
+
+
+def csc_domains_configuration_list_command(client: Client, args) -> CommandResults:
+    """
+    Returning a list of domains configurations with the applied filters
+
+    Args:
+        client: CSCDomainManager client
+        args: demisto.args()
+
+    Returns:
+        A list of domains configurations with the applied filters
+    """
+    args_copy = copy.deepcopy(args)
+    args_copy['page'] = args_copy.get('page') or DEFAULT_PAGE_SIZE_CONFI
+    args_copy['page_size'] = args_copy.get('page_size') or DEFAULT_PAGE_SIZE_SEARCH
+
+    if args_copy.get('limit'):
+        args_copy['page'] = '1'
+        args_copy['page_size'] = args_copy.get('limit')
+
+    params_results = create_params_string(args_copy)
+    configurations_results = client.send_get_request(url_suffix="/domains/configuration", params=params_results)
+
+    configurations_list = configurations_results.get('configurations', [])
+    configurations_with_required_fields = get_domains_configurations_hr_fields(configurations_list)
+
+    results = CommandResults(
+        readable_output=tableToMarkdown('Filtered Configurations',
+                                        configurations_with_required_fields,
+                                        headers=HR_HEADERS_FOR_DOMAIN_CONFI_LIST),
+        outputs_prefix='CSCDomainManager.Domain.Configuration',
+        outputs_key_field='CSCDomainManager.Domain.Configuration.Domain',
+        outputs=configurations_list
+    )
+    return results
+
+
+def domain(client: Client, args, reliability) -> CommandResults:
+    """
+    Gets the domain
+
+    Args:
+        client: CSCDomainManager client
+        args: demisto.args()
+        reliability: The source reliability. Default set to A.
+
+    Returns:
+       the domain information
+    """
+    qualified_domain_name = args.get('domain')
+    domain_json = client.send_get_request(url_suffix=f"/domains/{qualified_domain_name}", params="")
+
+    dbot_score = Common.DBotScore(
+        indicator=qualified_domain_name,
+        indicator_type=DBotScoreType.DOMAIN,
+        integration_name="CSCDomainManager",
+        score=Common.DBotScore.NONE,
+        reliability=reliability
+    )
+
+    domain_context = Common.Domain(
+        domain=domain_json.get('domain'),
+        creation_date=domain_json.get('registrationDate'),
+        domain_idn_name=domain_json.get('idn'),
+        expiration_date=domain_json.get('registryExpiryDate'),
+        name_servers=domain_json.get('nameServers'),
+        registrant_name=get_whois_contacts_fields_for_domain(domain_json.get('whoisContacts'),
```

Comment:
extract domain_json.get('whoisContacts') to a variable to make each line a bit shorter and easier to read